apps/dashboard/priv/repo/seeds.exs

```diff
@@ -38,6 +38,20 @@ components = [
     module: "DashboardWeb.Components.CheckIns",
     name: "Check-ins",
     refresh_type: "poll"
+  },
+  %{
+    api_path: "/check-ins/v2/check_ins?order=updated_at&filter=regular&per_page=500&offset=4850",
```

Comment:
this is a strange set of params

Reply:
Yeah - it was to get me a number that would actually update when I checked someone in.

Reply:
If/when we have access to meta data, this would switch to use that instead.

Reply:
We've got it now! 🎊 

https://github.com/geolessel/pco-dashboard/blob/57d8562521b9a21e7fe2125252e32c3c34e098df/apps/dashboard/lib/dashboard/stores/component_store.ex#L125-L138

You can get to it by doing something like this (untested code incoming):

```elixir
response = Dashboard.Stores.get(genserver_id(assigns), :last_response)
response["meta"]
```